bucket/gitql.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.6.0",
+    "description": "SQL like lanuage to perform queries on .git files",
+    "homepage": "https://github.com/amrdeveloper/gql",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/AmrDeveloper/GQL/releases/download/0.6.0/gql-x86_64-windows.exe",
+            "hash": "0b7dc2b055204d12eae9f8e29f2755d7df1a63f527af10340a489b42328b6e3f"
+        }
+    },
+    "pre_install": "Rename-Item \"$dir\\gql-x86_64-windows.exe\" \"$dir\\gitql.exe\"",
```

Comment:
```suggestion
```

Unnecessary if `#/` operator used to rename.

Reply:
Done, Thanks for suggestion @aliesbelik